airbyte-cdk/java/airbyte-cdk/db-sources/src/testFixtures/java/io/airbyte/cdk/testutils/TestDatabase.java

```diff
@@ -99,8 +99,8 @@ public T with(String fmtSql, Object... fmtArgs) {
   final public T initialized() {
     inContainerBootstrapCmd().forEach(this::execInContainer);
     this.dataSource = DataSourceFactory.create(
-        getUserName(),
-        getPassword(),
+        getContainer().getUsername(),
+        getContainer().getPassword(),
```

Comment:
We should avoid using `root` or `admin` or whatever shared super-user account is referenced here. Instead, use the credentials which are owned by this instance and which are created in the line above.

Reply:
the problem with those credentials is that this change is limiting them to be SSL-only, which didn't work for some reason (don't remember the details)